app/plugin.go

```diff
@@ -182,6 +189,78 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 	a.SyncPluginsActiveState()
 }
 
+// SynchPlugins will synchronize the plugins installed locally
+// with the plugin bundles available in the file store.
+func (a *App) SynchPlugins() *model.AppError {
+	mlog.Info("Synching plugins with the file store ...")
```

Comment:
nit: I'd drop the trailing `...`
```suggestion
	mlog.Info("Synching plugins with the file store")
```